app/lib/govuk/markdown_renderer.rb

```diff
@@ -0,0 +1,36 @@
+module Govuk
+  class MarkdownRenderer < ::Redcarpet::Render::Safe
+    def table(header, body)
```

Comment:
@dankmitchell  just to be sure, could we have achieved this by styling tables in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/app/frontend/styles/_styled-content.scss?

Reply:
Yes but it’s not a mixin so would have involved copying all the SCSS into the app and then any upgrades to govuk-frontend might be slightly messier.

I chose to extend markdown instead, we’ve done the same on FIND